alf/algorithms/actor_critic_algorithm.py

```diff
@@ -152,6 +151,8 @@ def calc_loss(self, training_info, final_time_step, final_policy_step):
 
             training_info = training_info._replace(
                 reward=training_info.reward + training_info.info.icm_reward)
+            final_time_step = final_time_step._replace(
+                reward=final_policy_step.info.icm_reward)
```

Comment:
why is not 

     final_time_step = final_time_step._replace( 
              reward= final_time_step.reward+final_policy_step.info.icm_reward)